front/src/services/__tests__/extractDataFromSearchParam.ts

```diff
@@ -0,0 +1,49 @@
+import "@testing-library/jest-dom";
+import { extractDataFromSearchParam } from "../helpers/formatSearchParamToPrismaQuery";
+
+describe("extractDataFromSearchParam", () => {
+    it("should return an object with the default values if no search params are passed", () => {
+        const { timeframe, sort, searchCondition } =
+            extractDataFromSearchParam();
+
+        const difference = timeframe.lte.getTime() - timeframe.gte.getTime();
+        expect(difference).toEqual(3600000);
+        expect(sort).toEqual({ datetime_utc: "desc" });
+        expect(searchCondition).toEqual([
+            {
+                input_string: {
+                    contains: "",
```

Comment:
does [prisma undefined](https://www.prisma.io/docs/concepts/components/prisma-client/null-and-undefined) not apply here?

Reply:
Replacing undefined with an empty string made it possible to handle everything as a string. I'll add a commit to handle "undefined" => we will have conditional statements here and there in the formatting.